bot/Exchanges/Binance.py

```diff
@@ -668,14 +687,13 @@ def updateSQLOrderModel(self, order, new_order_response, bot):
 		new_order_response. Should be part of the exchange interface  """
 
 		if order.is_test:
-			order.take_profit_price = self.toValidPrice(
-				symbol = order.symbol,
-				desired_price = Decimal(order.entry_price) * (Decimal(100) + Decimal(bot.profit_target))/Decimal(100), 
-				round_up=True)
+			order.entry_price = self.getCurrentTickPrice(order.symbol)['lastPrice']
+			if (order.order_type == self.ORDER_TYPE_MARKET) and not order.is_entry:
+				order.price = ((Decimal(100) - Decimal(bot.exit_settings.stop_loss_value))/Decimal(100)) * Decimal(order.entry_price)
 
 		else:
 			order.timestamp = new_order_response['transactTime']
-			order.entry_price = new_order_response['price']
+			order.price = new_order_response['price']
```

Comment:
Why are we only changing price here, and not entry_price as well?

Reply:
In line with the explanation I gave in the previous comments about entry_price we only update entry_price in updateSQLOrderModel if the order was a buy order. Otherwise we take entry price from the previous order. This should happen in botcontroller (Which I think I forgot).